src/planscape/rscripts/forsys.R

```diff
@@ -289,14 +289,24 @@ call_forsys <- function(
   forsys_inputs <- rbind(priorities, outputs)
   stand_data <- get_stand_data(connection, scenario, forsys_inputs)
 
+  if (length((configuration$weights)) == 0) {
+    print("using generated weights")
+    weights <- rep(1, length(priorities$condition_name))
+  } else {
```

Comment:
Can we also add a check if the number of provided weights ≠ the number of scenario priorities? (e.g either default the unaccounted for weights to 1 or just set them all to 1)